### ISSUE TITLE
Jails doesn't seem to complain when the config json is incorrect

### ISSUE THREAD

**Issue by valignatev** [SogoCZE/Jails#26]
I messed up and added a trailing comma in my jails.json file, and it took a while to figure out why jails won't respect the configuration, because I haven't seen anything in the logs about config not being valid. Seems like the server just silently applies the defaults if something goes wrong. Maybe a log entry about failing to parse the config file would be useful for situations like this.

**Comment by dbechrd** [comment]
I also had a fair bit of confusion trying to configure it correctly. It seems like VS code shows the default under Workspace settings, even when you click reset and it's supposed to be inheriting the path from User settings. Idk if this is a VS Code bug or what.. but eventually it started working. I think you also have to restart the extension every time you update the config for it to take effect, that seemed to help me.

**Comment by valignatev** [comment]
Yeah, restarting the extension a couple times and looking at debug output indicates when it gets the new settings.
